1021-radio-design-guidance.md

```diff
@@ -0,0 +1,25 @@
+---
+category: component design guidance
+state: draft
+created: 2024-10-16
+updated: 2024-10-21
+---
+
+# Radio Component
+
+[Radio](https://clarity.design/documentation/radio) component is a form element that allow the user to select one of the options in the list.
+
+## Guidance
+
+- **Must** use when only one item can be selected from a list.
+- **Must** NOT leave all radio buttons unselcted. Mark the first item or most preferred item as selected.
+- **Must** use when the list includes at least two mutually exclusive options.
+- **Must** NOT use when more than 6 options are available to choose from. Consider using a `select` component instead.
+- **Must** NOT place interactive content (buttons, links, etc) inside the label.
```

Comment:
This applies to all form control labels, not just radio buttons. Perhaps this should just be in the forms page.